internal/pkg/manifest/pipeline.go

```diff
@@ -187,8 +174,23 @@ func validateVersion(pm *pipelineManifest) (PipelineSchemaMajorVersion, error) {
 	}
 }
 
-// CFNTemplate serializes the manifest object into a CloudFormation template.
-func (m *pipelineManifest) CFNTemplate() (string, error) {
-	// TODO: #223 Generate CFN template for the archer pipeline
+func (m *PipelineManifest) StackName() string {
+	return m.ProjectName + "-" + m.Name
+}
+
+func (m *PipelineManifest) Template() (string, error) {
 	return "", nil
 }
+
+func (m *PipelineManifest) Parameters() []*cloudformation.Parameter {
+	return nil
+}
+
+func (m *PipelineManifest) Tags() []*cloudformation.Tag {
+	return []*cloudformation.Tag{
+		{
+			Key:   aws.String(archerCfn.ProjectTagKey),
+			Value: aws.String(m.ProjectName),
+		},
+	}
```

Comment:
Just want to note that ideally the `cloudformation.Parameter` and `cloudformation.Tag` constructs should not leak into clients IMO.